packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -11,84 +11,130 @@
-            </template>
-            <template if:true={activeParent}>
-              <li class="facet__active-parent slds-grid slds-grid_vertical-align-center slds-p-around_x-small slds-m-left_medium">
-                <span class="nowrap parent-text__bold">{activeParent.value}</span>
-                <span class="facet__number-of-results parent-text__bold slds-var-m-left_xx-small">({activeParent.numberOfResults})</span>
-              </li>  
-            </template>
-            <li class="slds-grid">
-              <ul class="slds-size_1-of-1">
+            <fieldset>
+              <legend class="slds-assistive-text">{field}</legend>
+              <ul>
+                <template if:true={hasParents}>
+                  <li class="facet__value-option slds-size_1-of-1 slds-p-around_x-small">
+                    <div
+                      class="facet__allCategories slds-grid slds-grid_vertical-align-center"
+                      role="button"
+                      aria-pressed="false"
+                      onclick={reset}
+                    >
+                      <lightning-icon
+                        class="slds-m-right_xxx-small"
+                        icon-name="utility:chevronleft"
+                        size="xx-small"
+                      ></lightning-icon>
+                      <span class="nowrap">{labels.allCategories}</span>
+                    </div>
+                  </li>
+                </template>
+
+                <template for:each={nonActiveParents} for:item="v">
+                  <li class={categoryFacetLiClass} key={v.value}>
+                    <c-quantic-category-facet-value
+                      onselectvalue={onSelectValue}
+                      item={v}
+                      non-active-parent="true"
+                    ></c-quantic-category-facet-value>
+                  </li>
+                </template>
+
+                <template if:true={activeParent}>
+                  <li
+                    class="facet__active-parent slds-grid slds-grid_vertical-align-center slds-p-around_x-small slds-m-left_medium"
+                    key={activeParent.value}
+                  >
+                    <c-quantic-category-facet-value
+                      onselectvalue={onSelectValue}
+                      item={activeParent}
+                    ></c-quantic-category-facet-value>
+                  </li>
+                </template>
+              </ul>
+              <ul>
                 <template for:each={values} for:item="v">
-                  <c-quantic-category-facet-value
-                    onselectvalue={onSelectValue} 
-                    key={v.value} 
-                    item={v}
-                    active-parent={activeParent}>
-                  </c-quantic-category-facet-value>
+                  <li class={categoryFacetLiClass} key={v.value}>
+                    <c-quantic-category-facet-value onselectvalue={onSelectValue} item={v}>
```

Comment:
Why did the `active-parent` go?

Reply:
The attribute active-parent was passed down to `<quantic-category-facet-value>` to generate `categoryFacetLiClass` for element `<li>`, and as  `<li>` is moved to <quantic-category-facet>, active-parent is no longer used in  `<quantic-category-facet-value>`, and   `<quantic-category-facet-value>` also has nonActiveParent  to store the same state.